packages/cli/bin/setup.js

```diff
@@ -11,7 +11,7 @@ const logger = Logger('setup')
 const { network, guardians: guardiansNumber, disputes } = yargs
```

Comment:
I didn't find documentation on this script in the package's README, so wasn't sure when this was meant to be used. This is just for setting up an environment on the local chain?